packages/react/src/useSynthql.ts

```diff
@@ -25,7 +25,7 @@ export function useSynthql<
 ): UseQueryResult<QueryResult<DB, TQuery>> {
     const { endpoint, requestInit } = useSynthqlContext();
 
-    const enrichedEndpoint = `${endpoint}/${query.from}`;
+    const enrichedEndpoint = `${endpoint}/${query.name ?? query.from}-${query.hash}`;
```

Comment:
Hmmm several issues here:
1. the hash might not be URL-safe
2. the hash (as it currently stands) can be _reaaaaally_ long.